.mci.yml

```diff
@@ -394,12 +419,12 @@ tasks:
         - func: "fetch mongodb"
```

Comment:
I sort of left it off intentionally, the idea being that we only explicitly name the storage engine when it differs from the default on that version.